README.md

```diff
@@ -10,4 +10,11 @@ mvn clean install
 
 ### Kontakt og spørsmål
 
-Opprett en issue i GitHub for eventuelle spørsmål.
\ No newline at end of file
+Opprett en issue i GitHub for eventuelle spørsmål.
+
+### Lokal kjøring
+1. Legg inn testdata i `dialog.sql` (se `TestApplication`). Det opprettes en tom "dialog" med ID 123 ved oppstart.
+1. Start `TestApplication.main`.
```

Comment:
Prøvde meg på denne lokalt. Fikk feilmelding "Could not find a valid Docker environment.". Kan du hjelpe meg eller oppdatere readme med det man må gjøre?

Reply:
Må ha kjørende en tilgjengelig Docker-instans for å bruke Testcontainers. Oppdaterer dok'en.

Reply:
Der ja, da fikk jeg den opp. Takk skal du ha